src/PrestaShopBundle/Resources/config/services/adapter/order_return.yml

```
@@ -21,10 +21,47 @@ services:
       - '@prestashop.adapter.order_return.repository.order_return_repository'
       - '@prestashop.adapter.order_return_state.repository.order_return_state_repository'
 
-  prestashop.adapter.order_return.repository.order_return_repository:
-    class: 'PrestaShop\PrestaShop\Adapter\OrderReturn\Repository\OrderReturnRepository'
+  PrestaShop\PrestaShop\Adapter\OrderReturn\Repository\OrderReturnRepository:
+    autowire: true
     arguments:
-      [ '@prestashop.adapter.order_return.validator.order_return_validator' ]
+      - '@doctrine.dbal.default_connection'
+      - '%database_prefix%'
+      - '@prestashop.adapter.order_return.validator.order_return_validator'
+
+  prestashop.adapter.order_return.repository.order_return_repository:
+    alias: PrestaShop\PrestaShop\Adapter\OrderReturn\Repository\OrderReturnRepository
```

```suggestion
    alias: PrestaShop\PrestaShop\Adapter\OrderReturn\Repository\OrderReturnRepository
    deprecated: ~
```

When you define aliases like this you should also add the deprecated config, this way we'll be able to remove them in a next major